mobify/hora.js
# Upgrade to specification (1.0.0)

Upgrading the interface to the latest spec (1.0.0, documented in the wiki).

Status: **Ready to merge**

Reviewers: @scalvert 
## Changes
- Most functions are now methods of a container (Hora.searchToggle -> Hora.search.toggle) (noun.verb).
- Event titles are now title case, not dashed (first-swipe -> First Swipe).
- Added action/value to some events.
- Add unit tests for all events.
- Bump to v1.0.0
## How to test-drive this PR
- checkout this branch
- `npm install`
- `grunt build`
- ensure file is built correctly
- `grunt test`
- ensure all unit tests pass correctly
